core/src/test/java/org/testcontainers/junit/wait/LogMessageWaitStrategyTest.java

```diff
@@ -0,0 +1,38 @@
+package org.testcontainers.junit.wait;
+
+import org.jetbrains.annotations.NotNull;
+import org.junit.Test;
+import org.testcontainers.containers.wait.LogMessageWaitStrategy;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * Tests for {@link LogMessageWaitStrategy}.
+ */
+public class LogMessageWaitStrategyTest extends AbstractWaitStrategyTest<LogMessageWaitStrategy> {
+
+    private static final String READY_MESSAGE = "I'm ready!";
+
+    @Test
+    public void testWaitUntilReady_Success() {
+        waitUntilReadyAndSucceed("while true; do sleep 1; echo -e \"" + READY_MESSAGE + "\"; done");
```

Comment:
I wanted a delay between the prints to avoid excessive printing, your version sounds better though.